texcount.py

```diff
@@ -55,8 +57,13 @@ def run(self, edit, **args):
             )
 
             result = p.communicate()[0].decode('utf-8').strip()
+            # result = p.communicate()[1].decode('utf-8').strip()
             if p.returncode == 0:
-                self.view.window().show_quick_panel(result.splitlines()[1:-4], None)
+                res_split = result.splitlines()
+                try:
+                    self.view.window().show_quick_panel(res_split[1:4] + res_split[9:], None)
+                except TypeError:
+                    self.view.window().show_quick_panel(res_split[1:4], None)
```

Comment:
Yes you are right it is because I was having problems calling texcount originally with `total` and getting a `TypeError` but it is redundant. I've pushed the changes removing the try and just using the top line.